packages/compass-aggregations/src/components/pipeline-results-workspace/pipeline-results-list.tsx

```diff
@@ -16,18 +16,20 @@ const containerStyles = css({
 
 const PipelineResultsList: React.FunctionComponent<{
```

Comment:
Can't leave a comment there directly, but the type of `DocumentJsonView` as defined in [index.d.ts here](https://github.com/mongodb-js/compass/blob/main/packages/compass-crud/index.d.ts#L3) is not correct, can you fix this while you are working a relevant part of the code: from how it's implemented `DocumentJsonView` only allows hadron document instances to be passed

Reply:
Added in [5ccb375](https://github.com/mongodb-js/compass/pull/3651/commits/5ccb375f88a69caa5ab56b55b3f353c5c42e9bb9)